tests/nosetests/pyanaconda_tests/module_subscription_tests.py

```diff
@@ -161,6 +165,10 @@ def setUp(self):
         self.callback = PropertiesChangedCallback()
         self.subscription_interface.PropertiesChanged.connect(self.callback)
 
+        # some of the diffs might be long if a test fails, but we still
+        # want to see them
+        self.maxDiff = None
+
     def _test_dbus_property(self, *args, **kwargs):
```

Comment:
Please rename this to `_check_dbus_property`().

Reply:
I did that. Though, looking at our tests, it looks like we are using both at the moment. I guess we might want to "standardise" one one if them in the future ?

Reply:
Yep, it's copied from somewhere. I have to change this globally everywhere to avoid another wrong copy.